### ISSUE TITLE
YDJ民科一句话读论文(2)： GaussianSplatting & DSS &  DPRF

### ISSUE THREAD

**Issue by yuedajiong** [graphdeco-inria/gaussian-splatting#454]
I am a scientist-guise layman (民科 in Chinese), my understanding is probably wrong.
the GS authors are very powerful, so I just compare a few papers:

1. GS: this paper
2. DSS: Differentiable Surface Splatting for Point-based Geometry Processing
3. DPRF: Differentiable Point-Based Radiance Fields for Efficient View Synthesis

DPRF: princeton
    shape: point , xyz-movable  (raw point, so can be rendered by pytorch3d PC renderer)
    color: learnable RGB SH coefficients per-point
    render:  differentable/e2e-ok;  rast, direct pytorch3d point-cloud raster & render
    tricks: coarse-to-fine strategy
    others: no lighting
    why-faster:    point-cloud-splating rending than MLP-ray-marching, no sampling

DSS: eth
    shape:  anisotropic ellipse, xyz-movable (not raw-point, 2d and normals), points-add/del, ..
    color: raw-rgb
    render: differentable/e2e-ok; extended, in CUDA;   performance-tricks: EWA
    tricks: some
    optimze:  gradients-based
    others:  with lighting

GS:
   shape: gauss-ball, xyz-movable, no normals but 3d-ball, scale dynamic, points-purne-clone, ...
   color: learnable SH coefficients per-point,  sh-degree(monotonic growth while training, in render, limited in code:2^4), powerful, more parameters
   render: differentable/e2e-ok; extended, in CUDA;   performance-tricks: tile
   tricks: lots of
   optimze:  gradients-based,  different-lr-for-different-parameters
   others: no lighting, diffcult to surface

if right and useful, please one-key-double-click-triple-income: like, favor, coin.   :-)
